src/RpcServer/RpcServer.SmartContract.cs

```diff
@@ -142,19 +149,14 @@ private static JObject ToJson(System.Collections.Generic.IEnumerable<DataCache.T
             return array;
         }
 
-        private static JObject ToJson(StackItem item, int max)
+        private static JObject ToJson(StackItem item, Session session)
```

Comment:
This implementation doesn't handle iterators inside of arrays or maps. Is that a scenario we consider important?

Reply:
No, it doesn't. Currently I don't see a need for this. We can add this support in the future if the need arises.